_rules/printable-characters-shortcut-ffbc54.md

```diff
@@ -0,0 +1,363 @@
+          <input id="remap2" type="checkbox" onclick="toggleModifier('secondShortcut', this.checked)">
+          Use "ctrl" key together with the "a" key
+        </label>
+      </div>
+    </div>
+    <br />
+    <div>
+      To do list
+    </div>
+    <ul id="list"></ul>
+  </body>
+</html>
+```
+
+#### Passed Example 4
+
+This [HTML document][] is listening to [keyboard events][keyboard event] with the attribute `key` being a [printable character][] and the method `getModifierState` returning `false` that cause [changes in content][]. There exists an [instrument][] to **remap** those [keyboard events][keyboard event] that cause [changes in content][] so that [same key][same key events] events are [blocked][blocked event] unless `getModifierState("Control")` returns `true`. In this example, the same [instrument][] is used to **remap** all [keyboard events][keyboard event].
+
+```html
+<html>
+  <head>
+    <title>Passed Example 4</title>
+    <script src="/test-assets/ffbc54/shortcut.js"></script>
+  </head>
+  <body onload="registerShortcut({id: 'firstShortcut', shortcutKey: '+'}); registerShortcut({id: 'secondShortcut', shortcutKey: 'a'}); activateShortcuts();">
+    <label for="target">Add to list (press "+" or "a" to add):</label>
+    <input type="text" id="target" />
+    <div>
+      <div>Remap shortcut</div>
+      <div>
+        <label>
+          <input id="remap" type="checkbox" onclick="toggleModifier('firstShortcut', this.checked); toggleModifier('secondShortcut', this.checked);">
+          Use "ctrl" key together with the "+" or "a" key
+        </label>
+      </div>
+    </div>
+    <br />
+    <div>
+      To do list
+    </div>
+    <ul id="list"></ul>
+  </body>
+</html>
+```
+
+#### Passed Example 5
+
+This [HTML document][] is listening to [keyboard events][keyboard event] with the attribute `key` being a [printable character][] and the method `getModifierState` returning `false` that cause [changes in content][], but the [events are blocked][blocked event] when no `widget` has [focus][].
+
+```html
+<html>
+  <head>
+    <title>Passed Example 5</title>
+    <script src="/test-assets/ffbc54/shortcut.js"></script>
+  </head>
+
+  <body onload="registerShortcut({shortcutKey: '+', focusOnly: true}); activateShortcuts();">
+    <label for="target">Add to list (press "+" to add):</label>
+    <input type="text" id="target" />
+    <br />
+    <div>
+      To do list
+    </div>
+    <ul id="list"></ul>
+  </body>
+</html>
+```
+
+#### Passed Example 6
+
+This [HTML document][] is listening to [keyboard events][keyboard event] with the attribute `key` being a [printable character][] and the method `getModifierState` returning `false` that cause [changes in content][]. There exists a [set of clearly labeled instruments][] to disable the [keyboard event][] or to **remap** the [keyboard event][] so that [same key][same key events] events are [blocked][blocked event] unless `getModifierState("Control")` returns `true`.
```

Comment:
I have travel to understand the first of these two sentences. An attempt to clarify follows:




```suggestion
This [HTML document][] is listening to [keyboard events][keyboard event] which attribute `key` is a [printable character][], which method `getModifierState` returns `false`, and which cause [changes in content][]. There exists a [set of clearly labeled instruments][] to disable the [keyboard event][] or to **remap** the [keyboard event][] so that [same key][same key events] events are [blocked][blocked event] unless `getModifierState("Control")` returns `true`.
```